frontend/src/views/ManureAndImports/ManureImportModal.tsx

```diff
@@ -88,7 +83,7 @@ export default function ManureImportModal({
   };
```

Comment:
Just realized that the way this is implemented, it will override existing uuids. Could we slip in a check whether or not to generate a new UUID when editing? Something like:

```js
const onSubmit = () => {
    handleSubmit({ ...formData, uuid: formData.uuid || crypto.randomUUID() });
    handleDialogClose();
  };
```